server/service/src/processors/transfer/requisition/test.rs

```diff
@@ -97,18 +120,22 @@ async fn requisition_transfer() {
                 RequisitionTransferTester::new(&request_store, &response_store, &item1, &item2);
 
             tester.insert_request_requisition(&ctx.connection);
-            // todo manual trigger
-            delay_for_processor().await;
+            // manually trigger because inserting the requisition doesn't trigger the processor
+            ctx.processors_trigger
+                .requisition_transfer
+                .try_send(())
+                .unwrap();
+            delayed_with_retries!(requisition_processed(&ctx.connection), "wait for processor");
             tester.check_response_requisition_not_created(&ctx.connection);
             delay_for_processor().await;
```

Comment:
Are these `delay_for_processor()` calls actually needed? An update should work when the previous check succeeded, shouldn't it?